python/hsfs/core/feature_monitoring_result_engine.py

```diff
@@ -99,13 +101,18 @@ def save_feature_monitoring_result(
             util.convert_event_time_to_timestamp(datetime.now()), -3
         )
 
+        reference_statistics_id = (
+            reference_statistics.id
```

Comment:
you can skip assigning a variable and directly use the ternary expression as an arg in the result constructor

Reply:
Yes, It seemed cleaner to me in this way. But I can change it.